include/seqan3/range/decorator/gap_decorator_anchor_set.hpp

```diff
@@ -67,10 +66,18 @@ namespace seqan3
  * Resolving random access requires logarithmic access into the set and inserting or removing a gap symbol additionally
  * entails updating all subsequent elements in the set to preserve correct cumulative sizes.
  *
+ * ### The seqan3::gap_decorator_anchor_set::iterator type
+ *
+ * \attention The iterator of the seqan3::gap_decorator_anchor_set does not model the
+ *            [Cpp17LegacyInputIterator](https://en.cppreference.com/w/cpp/named_req/InputIterator) requirements of the
```

Comment:
Have I written Cpp17LegacyInputIterator?
```suggestion
 *            [Cpp17InputIterator](https://en.cppreference.com/w/cpp/named_req/InputIterator) requirements of the
```